src/app/bug/bug-details/bug-details/bug-details.component.spec.ts

```diff
@@ -103,8 +99,8 @@ describe('BugDetailsComponent', () => {
         // find the bug being selected
         expect(spectator.component.bug.name).toEqual(bugName); // bug?.name
 
-        // UI label element
-        const nameLabel = spectator.fixture.debugElement.query(By.css('label[data-testid="name-label"]'));
+        // UI label element - LOOK INTO IT
+        const nameLabel = spectator.fixture.debugElement.query(By.css('label[data-testid="name-label"]')); // access native elements to access data inside the label
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/JohnnyGitH/BugEDex/pullRequest?prid=8891152)
```suggestion
        const nameLabel = spectator.fixture.debugElement.query(By.css("label[data-testid=\"name-label\"]")); // access native elements to access data inside the label
```